fastlane/Fastfile

```diff
@@ -239,7 +262,80 @@ end
       project_slug: 'simplenote-macos',
       dsym_path: dSYM_PATH,
     )
+  end
+
+  #############################################################################
+  # build_and_upload_app_store
+  # ---------------------------------------------------------------------------
+  # This lane builds the app and uploads it to the Mac App Store, optionally
+  # creating a GitHub release.
+  # ---------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_upload_app_store [create_github_release:<create github release>]
+  #
+  # Example:
+  # bundle exec fastlane build_and_upload_app_store
+  # bundle exec fastlane build_and_upload_app_store create_github_release:true
+  #############################################################################
+  lane :build_and_upload_app_store do | options |
+    # This allows code signing to work on CircleCI. It is skipped if this isn't
+    # running on CI.
+    #
+    # See https://circleci.com/docs/2.0/ios-codesigning/
+    setup_circle_ci
```

Comment:
For other projects we have this in the `before` hook. Maybe it doesn't matter though?

Reply:
Good point. I'll update in a followup PR. 

The only thing I'll say is that is seems wasteful to call it on every lane if when not in CI. But... it's a minor waste and if we think about the future where "everything happens in CI" then it makes sense.